src/v2/view-builder/views/ov/ResendView.js

```diff
@@ -4,7 +4,7 @@ import { SHOW_RESEND_TIMEOUT } from '../../utils/Constants';
 
 export default View.extend({
   //only show after certain threshold of polling
-  className: 'hide resend-ov-link-view',
+  className: 'hide resend-ov-link-view margin-top-30',
```

Comment:
Added some margin to avoid this issue 
![Screen Shot 2021-03-17 at 3 45 17 PM](https://user-images.githubusercontent.com/23267876/111835927-d429d680-88b2-11eb-9332-f858b9717c66.png)

Reply:
can we add the margin in this class instead `resend-ov-link-view`